src/settings/Api.ts

```diff
@@ -44,20 +44,29 @@ export class Api {
 			.then((data: OCSResult<Folder[]>) => Object.keys(data.ocs.data).map(id => data.ocs.data[id]));
 	}
 
+	// Returns all NC groups
 	listGroups(): Thenable<Group[]> {
-		const version = parseInt(OC.config.version, 10);
-		if (version >= 14) {
-			return $.getJSON(OC.linkToOCS('cloud', 1) + 'groups/details')
```

Comment:
Regular users can't use this API, but we want delegated admins to be able to retrieve groups